jumpscale/packages/admin/actors/admin.py

```diff
@@ -77,5 +79,74 @@ def set_explorer(self, explorer_type: str) -> str:
                 {"data": f"{explorer_type} is not a valid explorer type, must be 'testnet' or 'main'"}
             )
 
+    @actor_method
+    def list_identities(self) -> str:
+        identities = j.core.identity.list_all()
+        identity_data = {}
+        for identity_name in identities:
+            identity = j.core.identity.get(identity_name)
+            identity_data[identity_name] = identity.to_dict()
+            identity_data[identity_name]["instance_name"] = identity.instance_name
+            identity_data[identity_name].pop("__words")
+        return j.data.serializers.json.dumps({"data": identity_data})
+
+    @actor_method
+    def get_identity(self, identity_instance_name: str) -> str:
+        identity_names = j.core.identity.list_all()
+        if identity_instance_name in identity_names:
+            identity = j.core.identity.get(identity_instance_name)
+
+            return j.data.serializers.json.dumps(
+                {
+                    "data": {
+                        "instance_name": identity.instance_name,
+                        "name": identity.tname,
+                        "email": identity.email,
+                        "tid": identity.tid,
+                        "explorer_url": identity.explorer_url,
+                    }
+                }
+            )
+        else:
+            return j.data.serializers.json.dumps({"data": f"{identity_instance_name} doesn't exist"})
+
+    @actor_method
+    def set_identity(self, identity_instance_name: str) -> str:
+        identity_names = j.core.identity.list_all()
+        if identity_instance_name in identity_names:
+            j.core.identity.set_default(identity_instance_name)
+
+            # update our solutions
+            j.sals.reservation_chatflow.get_solutions_explorer()
```

Comment:
now `get_solutions_explorer()` just return the reservation data, to update local call `update_local_reservations()`

Reply:
updated here https://github.com/threefoldtech/js-sdk/pull/261/commits/9b893d50386d152363927bdd0f437f0062086ec3